packages/Webkul/Shop/src/Listeners/Customer.php

```diff
@@ -20,6 +20,11 @@ class Customer extends Base
     public function afterCreated($customer)
     {
         if (core()->getConfigData('customer.settings.email.verification')) {
+            try {
+                Mail::queue(new RegistrationNotification($customer, 'admin'));
+            } catch (\Exception $e) {
+                report($e);
+            }
```

Comment:
Here, we need to add a check for registration email notification as per configuration.